sdk/core/azure-core/src/test/java/com/azure/core/implementation/models/jsonflatten/JsonFlattenOnJsonIgnoredProperty.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.models.jsonflatten;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.JsonFlatten;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Model used for testing {@link JsonFlatten}.
+ */
+@Fluent
+public final class JsonFlattenOnJsonIgnoredProperty {
+    @JsonProperty("name")
+    private String name;
+
+    @JsonIgnore
+    @JsonFlatten
+    @JsonProperty("jsonflatten.ignored")
+    private String ignored;
```

Comment:
Is this even valid to have `@JsonIgnore` and `@JsonProperty`?

Reply:
Yes, but it doesn't really make much sense, I just wanted to ensure that `JsonFlatten` wouldn't result in an inappropriate serialization in this instance